### PR TITLE
ref(mypy): Silence type errors in snuba.redis

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: snuba/redis.py [hunk 1]
```diff
@@ -11,15 +11,15 @@
 RedisClientType = Union[StrictRedis, StrictRedisCluster]
 
 
-class RetryingStrictRedisCluster(StrictRedisCluster):
+class RetryingStrictRedisCluster(StrictRedisCluster):  # type: ignore
```
Comment (fpacifici) [inline snuba/redis.py:14]: mh why this ?

Comment (lynnagara) [reply snuba/redis.py:14]: The error is `Class cannot subclass 'StrictRedisCluster' (has type 'Any')`. Not sure if there's an alternate solution to fix this.

Comment (fpacifici) [reply snuba/redis.py:14]: ok, Please add a comment explaining why we silence that.

Comment (lynnagara) [reply snuba/redis.py:14]: sure

Comment (lynnagara) [reply snuba/redis.py]: Happens with a couple of our libraries with missing type hints, not sure if we want to try to do this in mypy.ini in future

**Review by fpacifici — APPROVED** [review]

**Pull request merged**